### DISCUSSION TITLE
Removed project but is still on the domain

### DISCUSSION THREAD

**Discussion by thecookieswhoplays** [vercel/community#2494] [Help] [answered]
Hi, So I tried to remove a project from vercel to use the domain somewhere else but I get this this error 
404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND
ID: iad1::rv4vg-1684381130416-824c8358a2a9
when going on the domain. I have even removed the domain from vercel

**Comment by amyegan** [comment]
Hi @thecookieswhoplays. Would you be able to share the domain here so I can look into this problem?

**Comment by gorkunov-m** [comment]
Similar problem, project removed but domain is still used ([project].vercel.app) and page available

**Reply by amyegan** [reply]
If you share the domain, I can look into this for you

**Reply by gorkunov-m** [reply]
https://remstanmach.vercel.app/

**Reply by amyegan** [reply]
There is currently a project associated with that domain. Did you transfer ownership to someone else?

**Reply by gorkunov-m** [reply]
No, there was an assumption that this is the same project, but with a slightly different name and located on a different account. Thanks, I'll figure it out

**Comment by nrishav007** [comment]
@amyegan I am also facing the same issue. can you please look into it? my domain is rarecombee.com
I am trying to deploy in AWS and got a CNAMEAlreadyExistsException error with distribution:*******YB3OR4K and account ID ******288780. Please resolve my issue ASAP.

**Reply by amyegan** [reply]
It looks like that domain is not associated with Vercel. Please work with your registrar and AWS 🙏

**Comment by GinaCastromonte** [comment]
I have the same issue with my project.  I removed the domain and when I go to the website I get this error : 404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND
ID: iad1::9chb6-1712858233534-d7c6c97347e4

**Reply by khalt00** [reply]
@GinaCastromonte I have the same problem, have you solved this?
Thanks

**Reply by amyegan** [reply]
Gina did find a solution. 🙂  You can read about it in https://github.com/vercel/community/discussions/6479

**Comment by khalt00** [comment]
Is this fix? or we have any solution for this?

**Comment by amyegan** [comment, marked as answer]
This is caused by DNS records. If your domain is still directed to Vercel servers, but you've deleted your project, you will see a [DEPLOYMENT_NOT_FOUND](https://vercel.com/docs/errors/DEPLOYMENT_NOT_FOUND) error page because our system doesn't have anywhere else to route the request. 

You'll need to update your DNS records if you would like the domain to take visitors to a different site. Please reach out to your registrar if you need help updating those records.
